administration/src/application/components/primitive-inputs/SelectForm.tsx

```diff
@@ -1,23 +1,27 @@
 import { FormControl, FormHelperText, InputLabel, MenuItem, Select } from '@mui/material'
-import { useContext, useState } from 'react'
+import { useContext, useEffect, useState } from 'react'
 import { Form } from '../../FormType'
 import { ShortTextInput } from '../../../generated/graphql'
 import { FormContext } from '../SteppedSubForms'
 
 type State = { selectedText: string }
 type ValidatedInput = ShortTextInput
-type Options = { items: string[] }
+
+export type SelectItem = { label: string; value: string }
+type Options = {
+  items: SelectItem[]
+}
 type AdditionalProps = { label: string }
 const SelectForm: Form<State, Options, ValidatedInput, AdditionalProps> = {
   initialState: { selectedText: '' },
   getArrayBufferKeys: () => [],
   validate: ({ selectedText }, options) => {
     if (selectedText.length === 0) return { type: 'error', message: 'Feld ist erforderlich.' }
-    if (!options.items.includes(selectedText))
-      return {
-        type: 'error',
-        message: `Wert muss einer der auswählbaren Optionen entsprechen.`,
-      }
+    // if (!options.includes(selectedText))
```

Comment:
why we need this?

Reply:
In principle, I think it is not necessary. It's just another layer of validation (in case our `selectedText` -- which should probably now be called `selectedValue` -- was somehow changed to an invalid string by MUI's Select element).

Reply:
If this is too much in your way, we can also drop it :)